packages/driver/src/cross-origin/events/socket.ts

```diff
@@ -1,18 +1,16 @@
-import { createWebsocket } from '@packages/socket/lib/browser'
-
 export const handleSocketEvents = (Cypress) => {
-  const webSocket = createWebsocket({ path: Cypress.config('socketIoRoute'), browserFamily: Cypress.config('browser').family })
-
-  webSocket.connect()
-
-  const onBackendRequest = (...args) => {
-    webSocket.emit('backend:request', ...args)
-  }
+  const onRequest = async (event, args) => {
```

Comment:
we now delegate any websocket connections to the primary cypress instance and communicate with it via post message.